common/src/main/java/com/ssblur/scriptor/item/Spellbook.java

```diff
@@ -135,7 +136,8 @@ public boolean render(
     int lightLevel
   ) {
     matrix.pushPose();
-    int v = hand == InteractionHand.MAIN_HAND ? 1 : -1;
+    boolean in_right_hand = hand == InteractionHand.MAIN_HAND ^ player.getMainArm() == HumanoidArm.LEFT;
```

Comment:
Can you change this to inRightHand to match codebase style? Otherwise, good change, will merge.